src/masonite/input/InputBag.py

```diff
@@ -56,11 +56,6 @@ def parse(self, environ):
                 self.post_data = self.parse_dict(parsed_request_body)
 
             elif "multipart/form-data" in environ.get("CONTENT_TYPE", ""):
-                try:
```

Comment:
not used

Reply:
hmm. How do you know this isn't used? We may just not have a test for it.

I would assume in the event the browser does not pass a content length for the request this would throw an error right?

Reply:
I mean that the variable in the code is really not used later

Reply:
Oh ok cool lol